proposals/stack-switching/Explainer.md

```diff
@@ -833,6 +833,126 @@ events and only `(on $e switch)` handlers can handle `switch`
 events. The handler search continues past handlers for the wrong kind
 of event, even if they use the correct tag.
 
+#### Store extensions
+
+* A store component `tags` for allocated tags (from the exception
```

Comment:
Since the exception handling proposal is already merged, I think we can consider taginsts and their store component given.